Sources/OuterCore/Value.swift

```diff
@@ -9,7 +9,8 @@ import Foundation
 
 public class Value: Hashable {
   let name: String
-  unowned var type: Value
+  var type: Value
```

Comment:
This can remain unowned as long as the type table in the module isn’t destroyed before the value is.  I ran into that consistently while working with the parser.